server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlVisitor.java

```diff
@@ -112,6 +112,33 @@ public List<Node> visitResult_set_locator_host_variable(
     return ImmutableList.of(variableDefinitionNode);
   }
 
+   @Override
+   public List<Node> visitBinary_host_variable(Db2SqlParser.Binary_host_variableContext ctx) {
```

Comment:
Doc suggest this VARBINARY would generate source member as mentioned  below
![image](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/assets/69206564/f801aff9-f5ed-4fa0-b456-e5405ef822b4)

This would mean insert new variables. We can refer below sample code , which has the same use case

https://github.com/BroadcomMFD/cobol-dialect-template/blob/develop/server/dialect-example/src/main/java/org/broadcom/cobol/dialects/cobolExample/nodes/Level100Node.java

https://github.com/BroadcomMFD/cobol-dialect-template/blob/develop/server/dialect-example/src/main/java/org/broadcom/cobol/dialects/cobolExample/processor/Level100Transformer.java

https://github.com/BroadcomMFD/cobol-dialect-template/blob/fe28bea4382cd42b9a0c93b60e7e473610c03b18/server/dialect-example/src/main/java/org/broadcom/cobol/dialects/cobolExample/ExampleDialect.java#L201

Reply:
Also, we would need to add test, for usage of generated variables

Reply:
The IBM DB2 preprocessor converts the DB2 variable types to those supported by COBOL. These generated variables should already be taken care of.

Reply:
The idea is that below program with db2 pre/co-processor would compile fine. But cobol-ls would start to throw error variable not defined. 
```cobol
        Identification Division.
        Program-Id. 'TEST1'.
        Data Division.
         Working-Storage Section.
         LINKAGE SECTION.
         01 VBIN-VAR USAGE IS SQL TYPE IS BINARY VARYING(10).
        PROCEDURE division.
           display  VBIN-VAR-LEN.
           display  VBIN-VAR-TEXT.

```

Reply:
I'll take it from here. Let's merge this PR if everything else is fine.